docs/troubleshooting/different-private-key.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: Different Private Keys across Integrations
+image: "images/docs-meta-cards/documentation-card.png"
+displayed_sidebar: docs
+description: "Different Private Keys across Integrations | Documentation - Web3Auth"
+---
+
+Users frequently face confusion when they get different wallet addresses due to multitudes of reasons. Here's our attempt to explain the possible
+reasons for different wallet addresses and what to keep in mind while designing your solutions.
+
+## Factors affecting wallet addresses
+
+### Verifier
+
+A verifier is a piece of information about the OAuth provider being used by the application, which is queried by web3auth auth network nodes from a
+smart contract deployed on the Ethereum blockchain to verify the JWT token. To know more about creating verifiers, please refer
+[here](/docs/auth-provider-setup/verifiers).
+
+As the verifiers change, the private key changes. And as we go across different SDKs, we need different verifiers to be created, hence changing the
+wallet addresses for the end-users.
+
+### Aggregate Verifier
```

Comment:
Make this a sub section inside the verifier point. Restructure according to the comments above.

Mark Aggregate verifier as a resolution to the problem rather